.github/workflows/pypi-release.yml

```diff
@@ -9,17 +9,18 @@ jobs:
   build-and-publish:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v3
```

Comment:
Why is this one different from the other changes? (v4)

Reply:
A silly mistake. I'll update to v4.